cql3/functions/functions.cc

```diff
@@ -332,6 +332,9 @@ functions::get(data_dictionary::database db,
         if (!receiver_cf.has_value()) {
             throw exceptions::invalid_request_exception("functions::get for token doesn't have a known column family");
         }
+        if (schema == nullptr) {
+            throw exceptions::invalid_request_exception(format("functions::get for token cannot find {} table", *receiver_cf));
+        }
```

Comment:
mutation_fragments(tab) should have a pseudo-schema. Oh well.

Reply:
/cc @denesb

Reply:
It has pseudo-schema, but it's temporary and not find-able via database/data_disctionary, thus this issue

Reply:
@denesb , #18759 has the test for that